data/decks/noel.json

```diff
@@ -0,0 +1,179 @@
+{
+	"id": "noel",
+	"season": 5,
+	"display_name": "Noel Vermillion",
+	"cardback": "cardback_noel.jpg",
+	"exceed_cost": 4,
+	"exceed_to_overdrive": true,
+	"on_exceed": {
+		"effect_type": "pass"
+	},
+	"overdrive_effect": {
+		"effect_type": "choice",
+		"choice": [
+			{ "effect_type": "close", "amount": 1 },
+			{ "effect_type": "retreat", "amount": 1 }
+		]
+	},
+	"ability_effects": [
+		{
+			"timing": "on_reshuffle",
+			"condition": "manual_reshuffle",
+			"effect_type": "add_set_aside_card_to_deck",
+			"id": "noel_valkyrieveil"
+		},
+		{
+			"timing": "set_strike",
+			"condition": "initiated_strike",
+			"effect_type": "gauge_for_effect",
+			"per_gauge_effect": null,
+			"overall_effect": {
+                "character_effect": true,
```

Comment:
probably doesn't matter a huge amount, but it may be nice to line up the indentation here and in the exceed ability effects (unless that's just a github formatting thing?)

Reply:
think this one is a github formatting thing, not seeing it in my own source files